create_containers.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2154,SC2143,SC2034,SC1091
+set -euo pipefail
+
+source ./docker_functions.sh
+
+# Create networks for containers
+log "Checking for existing networks..."
+for network in "${networks[@]}"
+do
+  if [ "$(docker network ls | grep -w "${network}")" ]; then
+    log "Network ${network} exists, skipping..."
+  else
+    OUTPUT=$(networks)
```

Comment:
The naming becomes confusing here, should probably be `create_networks()` or something like that to signify what happens. Same for some of the other functions I think.

Also $OUTPUT is never used?

Reply:
I have fixed my naming convention, but I'm not sure I follow your comment regarding OUTPUT never being used:
`oliver@skynet:~/reload/onboarding-oliver$ bash create_containers.sh 
09:52:25 Checking for existing networks...
09:52:25 Network todo-app created
oliver@skynet:~/reload/onboarding-oliver$ bash create_containers.sh 
09:53:42 Checking for existing networks...
09:53:42 Network todo-app exists, skipping...`